components/organisms/messaging/components/send/messagingSend.js

```diff
@@ -155,6 +155,7 @@ const MessagingSend = ({ audienceItems, maxLength, mentions, onSubmit }) => {
             ref={fileInputRef}
             style={{ display: 'none' }}
             type="file"
+            accept=".pdf,image/*"
```

Comment:
@Ojaghi1991 I will approve this but it really should be a prop I think in future.
We will allow videos and audio etc soon